protocols/migration/participants.py

```diff
@@ -7,6 +7,60 @@
 from protocols.migration import BaseMigration
 
 
+class MigrationParticipants100To104SNAPSHOT(BaseMigration):
+    old_model = participant_1_0_0
+    new_model = participant_1_0_4_SNAPSHOT
+
+    def migrate_cancer_participant(self, cancer_participant):
+        migrated_participant = self.new_model.CancerParticipant.fromJsonDict(cancer_participant.toJsonDict())
+
+        migrated_participant.yearOfBirth = 0
+
+        if cancer_participant.primaryDiagnosisDisease is not None:
+            migrated_participant.primaryDiagnosisDisease = [cancer_participant.primaryDiagnosisDisease]
+
+        if cancer_participant.primaryDiagnosisSubDisease is not None:
+            migrated_participant.primaryDiagnosisSubDisease = [cancer_participant.primaryDiagnosisSubDisease]
+
+        if cancer_participant.assignedICD10 is not None:
+            migrated_participant.assignedICD10 = [cancer_participant.assignedICD10]
+
+        migrated_participant.tumourSamples = self.migrate_tumour_samples(cancer_participant.tumourSamples)
+        migrated_participant.germlineSamples = self.migrate_germline_samples(cancer_participant.germlineSamples)
+
+        return self.validate_object(
+            object_to_validate=migrated_participant, object_type=self.new_model.CancerParticipant
+        )
+
+    def migrate_tumour_sample(self, tumour_sample):
+        migrated_tumour_sample = self.new_model.TumourSample.fromJsonDict(tumour_sample.toJsonDict())
+
+        migrated_tumour_sample.LDPCode = ''
+        migrated_tumour_sample.tumourId = str(tumour_sample.tumourId)
+        migrated_tumour_sample.diseaseType = tumour_sample.tumourType
+        migrated_tumour_sample.diseaseSubType = tumour_sample.tumourSubType
+        migrated_tumour_sample.tumourType = tumour_sample.phase
+
+        return self.validate_object(
+            object_to_validate=migrated_tumour_sample, object_type=self.new_model.TumourSample
+        )
+
+    def migrate_tumour_samples(self, tumour_samples):
+        return [self.migrate_tumour_sample(tumour_sample=tumour_sample) for tumour_sample in tumour_samples]
+
+    def migrate_germline_sample(self, germline_sample):
+        migrated_germline_sample = self.new_model.GermlineSample.fromJsonDict(germline_sample.toJsonDict())
+
+        migrated_germline_sample.LDPCode = ''
```

Comment:
LDPCode in CancerParticipant-1.0.0 may be fit within LDPCode under GermlineSample-1.0.4-SNAPSHOT ??? @Antonior26 ?

Reply:
This will have the participant LDPCode.
I'll make sure to add an explicit test @priesgo